src/directives/ClickToAccept.tsx

```diff
@@ -173,30 +182,45 @@ const ClickToAccept = ({ directive, status, mutate }: DirectiveProps) => {
                         }
                         onChange={handlers.update}
                         name="accept"
-                        label={
-                            <FormattedMessage
-                                id="legal.docs.accept"
-                                values={{
-                                    privacy: (
-                                        <FormattedMessage id="legal.docs.privacy" />
-                                    ),
-                                    terms: (
-                                        <FormattedMessage id="legal.docs.terms" />
-                                    ),
-                                }}
-                            />
-                        }
+                        label={intl.formatMessage(
+                            { id: 'legal.docs.accept' },
+                            {
+                                privacy: intl.formatMessage({
+                                    id: 'legal.docs.privacy',
+                                }),
+                                terms: intl.formatMessage({
+                                    id: 'legal.docs.terms',
+                                }),
+                            }
+                        )}
                     />
                 </FormControl>
 
-                <SafeLoadingButton
-                    type="submit"
-                    variant="contained"
-                    loading={saving}
-                    disabled={saving}
+                <Toolbar
+                    disableGutters
+                    sx={{
+                        justifyContent: 'space-between',
+                        width: '100%',
+                    }}
                 >
-                    <FormattedMessage id="cta.continue" />
-                </SafeLoadingButton>
+                    <Button
+                        disabled={saving}
+                        variant="outlined"
+                        onClick={async () => {
+                            await supabaseClient.auth.signOut();
+                        }}
+                    >
+                        {intl.formatMessage({ id: 'cta.cancel' })}
+                    </Button>
+                    <SafeLoadingButton
+                        type="submit"
+                        variant="contained"
+                        loading={saving}
+                        disabled={saving}
+                    >
+                        {intl.formatMessage({ id: 'cta.continue' })}
+                    </SafeLoadingButton>
+                </Toolbar>
```

Comment:
Should this be a component?

Reply:
I forgot the `form` submits the thing - I'll make this one a component real quick.